spatialpandas/geometry/polygon.py

```diff
@@ -35,7 +37,6 @@ def construct_array_type(cls):
 
     @classmethod
     def _shapely_to_coordinates(cls, shape):
-        import shapely.geometry as sg
```

Comment:
This is likely here to not make shapely a required dependencies.

Reply:
Got it! Initially I presumed that this might lead to _some_ slowdown, but just testing it and it doesn't lead to any.